force-app/main/default/objectTranslations/WorkOrderLineItem-no/workorderlineitem_startdate.fieldTranslation-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomFieldTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <article>None</article>
+        <plural>false</plural>
+        <value>Bestilt starttid</value>
+    </caseValues>
+    <gender>Masculine</gender>
+    <name>workorderlineitem_startdate</name>
+</CustomFieldTranslation>
```

Comment:
Det kan hende at oversettelse av standard felter skal ligge i unpackaged. Ser at det er litt spredt.
Hører med Carl, men vi lar det ligge her for nå.